WORKSPACE

```diff
@@ -253,6 +260,10 @@ fetch_shfmt()
 
 fetch_terraform()
 
-load("//.aspect/workflows:deps.bzl", "fetch_workflows_deps")
+load("@aspect_rules_workflows//workflows:dependencies.bzl", "rules_workflows_dependencies")
+
+rules_workflows_dependencies()
+
+load("@aspect_rules_workflows//workflows:repositories.bzl", "fetch_workflows_tools")
 
-fetch_workflows_deps()
+fetch_workflows_tools(workflows_version = "5.9.2")
```

Comment:
should rules_workflows be versioned with workflows, so you don't have to choose two separate versions when installing this?